UI/DropsController.gd

```diff
@@ -108,6 +122,36 @@ func _on_DropsTableItemsPanel_dropped_item(drop_item: GatheringItem):
 	_selected_drops_table.add_item(drop_item)
 	print_debug("Placed %s in drops table %s" % [tr(drop_item.item.name), _selected_drops_table.name])
 
+func _on_DropsFilterLineEdit_text_changed(new_text):
+	_FilterList(new_text)
+
+func _FilterList(filter_text: String = "") -> void:
+	var has_filter_text = !filter_text.empty()
+	
+	var filtered_tables: Array = []
+	var selected_table: DropsTable = _selected_drops_table
+
+	if selected_table != null and (selected_table.name.to_lower().find(filter_text.to_lower()) == -1):
+		filtered_tables.append(selected_table)
+
+	for drops_table in SetProvider.get_all_drops_tables():
+		var table_name_lower = drops_table.name.to_lower()
+		if filter_text.empty() or table_name_lower.find(filter_text.to_lower()) != -1:
+			filtered_tables.append(drops_table)
+
+	if $DropsFilterLineEdit != null:  # Check if $DropsFilterLineEdit is not null
```

Comment:
Why? I think it's safe to assume it exists, but even if it doesn't, I don't see why that would affect clearing